packages/core/docs/.vuepress/styles/index.styl

```
@@ -31,3 +30,69 @@ a code {
   margin: 20pt auto;
   box-shadow: 0 2px 6px 0 hsla(0, 0%, 0%, 0.2);
 }
+
+aside.sidebar a.sidebar-link.active {
+  font-weight: inherit;
+}
+
+.theme-default-content:not(.custom) a {
+  display: inline-flex;
+  color: inherit;
+  font-weight: inherit;
+  text-decoration: underline;
+  text-underline-offset: 3px;
+  text-decoration-thickness: 2px;
+  text-decoration-color: #22c34b60;
+}
+
+.theme-default-content:not(.custom) a:hover {
+  color: #22c34b;
+  text-decoration: underline;
+  text-underline-offset: 3px;
+  text-decoration-thickness: 2px;
+  text-decoration-color: #22c34b70;
+}
+
+q {
+  position: relative;
+  display: block;
+  margin: 30px 0;
+  padding: 40px 20px;
+  border-radius: 8px;
+}
+
+@media (max-width: 959px) {
+  q {
+    margin: 20px 0;
+    padding: 40px 0;
+  }
+}
+
+q::before,
+q::after {
+  position: absolute;
+  font-size: 40px;
+  line-height: 1;
+  color: #9CA3AF;
+}
+
+
+q::before {
+  content: '❝';
+  top: 5px;
+  left: 0;
+}
+
+q::after {
+  content: '❞';
+  bottom: -5px;
+  right: 0;
+}
+
+q p {
+  margin: 0px;
+  font-size: 1.1rem;
+  font-weight: 500;
+  font-style: italic;
+  text-align: center;
+}
```

just small suggestion

```suggestion
}

```